### ISSUE TITLE
Bootstrap @chrisvfritz

### ISSUE THREAD

**Issue by hessnd** [hessnd/tc359_rails_final#1]
I have added bootstrap into my project but i'm having trouble getting the javascript to work.  I have put in one of the responsive navbars but the dropdown menu button doesn't work when I make the page width smaller.  I thought I setup the javascript correctly, but could you help me figure out what i'm doing wrong? Thanks, Nick

**Comment by hessnd** [comment]
@chrisvfritz

**Comment by chrisvfritz** [comment]
I don't see where you're using the responsive navbar. Could you point me in the right direction?

**Comment by hessnd** [comment]
Im using the bootstrap grid system and a couple other of their components.  To get the point do we have to use the navbar?

**Comment by chrisvfritz** [comment]
No, you don't have to use a navbar - I was just trying to help you diagnose the problem. Are you still having a problem with the JavaScript?

**Comment by hessnd** [comment]
Nope I got it working

**Comment by chrisvfritz** [comment]
Cool - just let me know when this is done then.
